src/cameras/src/pm/Camera.cpp

```diff
@@ -697,59 +760,38 @@ bool pm::Camera<F>::setExp(const ExpSettings& settings) {
             return false;
     }
 
-    uint32_t exposeModeOut;
-    if (PV_OK != pl_get_param(ctx->hcam, PARAM_EXPOSE_OUT_MODE, ATTR_CURRENT, (void*)&exposeModeOut)) {
-        spdlog::error("EXPOSE_MODE_OUT could not be read, ({})", GetError());
-        return false;
-    }
-    spdlog::info("Expose mode out set to {}", exposeModeOut);
+    return true;
+}
 
-    uint32_t triggerMode;
-    if (PV_OK != pl_get_param(ctx->hcam, PARAM_EXPOSURE_MODE, ATTR_CURRENT, (void*)&triggerMode)) {
-        spdlog::error("EXPOSESURE_MODE could not be read, ({})", GetError());
-        return false;
-    }
-    spdlog::info("Trigger mode set to {}", triggerMode);
 
-    if (ctx->bufferBytes) { //need to deallocate old buffers first
-        ctx->frames.clear();
-        ctx->buffer.reset();
 
-        ctx->bufferBytes = 0;
-        ctx->buffer = nullptr;
+template<FrameConcept F>
+bool pm::Camera<F>::setExp(const ExpSettings& settings) {
```

Comment:
I'd rename this one to `updateExp` or `updateExpSettings` so the `setupContExp` can be switched back to `setExp` since this method calling the `setup_exp` method on the camera